tests/PHPStan/Analyser/data/filter-input.php

```diff
@@ -0,0 +1,36 @@
+<?php declare(strict_types=1);
+
+namespace FilterInput;
+
+use function PHPStan\Testing\assertType;
+
+class FilterInput
+{
+
+	public function invalidTypesOrVarNamesAreIgnored($mixed): void
+	{
+		assertType('mixed', filter_input(INPUT_GET, $mixed, FILTER_VALIDATE_INT));
+		assertType('mixed', filter_input(INPUT_GET, 17, FILTER_VALIDATE_INT));
+		assertType('mixed', filter_input(-1, 'foo', FILTER_VALIDATE_INT));
```

Comment:
fyi things like that lead to
```
Fatal error: Uncaught ValueError: filter_input(): Argument #1 ($type) must be an INPUT_* constant in /in/47GPB:3
```
which is not caught by phpstan yet. not sure if needed, but we could create a rule for it I suppose

https://3v4l.org/D6eor
https://phpstan.org/r/235a242d-c6b3-4a8f-a746-a766b4a9c687

Reply:
will be handled by returning a `*NEVER*` at least